### PR TITLE
Merger in copy-cross-lpar enhancements

### PR DESCRIPTION
**Pull request by KevinLoesch1**

### CODE REVIEW

File: packages/cli/src/zosfiles/copy/dsclp/Dsclp.handler.ts
Comment (gejohnston) [inline packages/cli/src/zosfiles/copy/dsclp/Dsclp.handler.ts:61]: Minor nit: Years ago, Tech Writing & QA had a guideline that we use the word "want" instead of "wish". Not a request for change, just something to consider.

File: packages/zosfiles/src/methods/copy/Copy.ts
Comment (gejohnston) [inline packages/zosfiles/src/methods/copy/Copy.ts:181]: We had team conversations about tagging (and loss of tagging) if a binary transfer is specified. Does that concern apply to datasets? If tagging is applicable, would it mean that the target would not have the same tag?

**Review by gejohnston — APPROVED** [review]
I posed a few questions that will not necessarily result in any request for change.

File: packages/cli/src/zosfiles/copy/Copy.options.ts
Comment (KevinLoesch1) [reply packages/cli/src/zosfiles/copy/Copy.options.ts]: This was a early change that should have been removed. I have deleted it from the project.

File: packages/cli/src/zosfiles/copy/dsclp/Dsclp.handler.ts
Comment (KevinLoesch1) [reply packages/cli/src/zosfiles/copy/dsclp/Dsclp.handler.ts]: This was an early change that is no longer needed. I have removed it from the project.

File: packages/zosfiles/src/methods/copy/Copy.ts
Comment (t1m0thyj) [inline packages/zosfiles/src/methods/copy/Copy.ts:138]: I believe the `console == undefined` check is left behind from when this method accepted a `console` parameter.

We should remove it now - it will always evaluate to `false` because the JS console API is always defined.

File: packages/zosfiles/src/methods/copy/Copy.ts
Comment (t1m0thyj) [reply packages/zosfiles/src/methods/copy/Copy.ts:181]: I believe tagging to specify encoding only applies to USS files.

**Review by t1m0thyj — CHANGES_REQUESTED** [review]

File: packages/zosfiles/src/constants/ZosFiles.messages.ts
Comment (KevinLoesch1) [reply packages/zosfiles/src/constants/ZosFiles.messages.ts:656]: I've made the change. It is now spelled consistently.

File: packages/cli/__tests__/zosfiles/__system__/copy/dsclp/__snapshots__/cli.files.copy.dsclp.system.test.ts.snap
Comment (KevinLoesch1) [reply packages/cli/__tests__/zosfiles/__system__/copy/dsclp/__snapshots__/cli.files.copy.dsclp.system.test.ts.snap:1]: I've made the suggested changes.

File: packages/zosfiles/src/methods/copy/Copy.ts
Comment (KevinLoesch1) [reply packages/zosfiles/src/methods/copy/Copy.ts:138]: This check has been removed.

File: packages/zosfiles/src/methods/copy/Copy.ts
Comment (t1m0thyj) [inline packages/zosfiles/src/methods/copy/Copy.ts:264]: If the default behavior of the API is to _not_ replace files, should we remove this condition? `overwriteTarget == undefined`

**Review by t1m0thyj — COMMENTED** [review]
Thanks for addressing earlier feedback, left one more question.

Will approve once tests are passing and merge conflicts are resolved 🙂

File: packages/zosfiles/src/methods/copy/Copy.ts
Comment (KevinLoesch1) [reply packages/zosfiles/src/methods/copy/Copy.ts]: I removed the check, as it is no longer relevant.

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @KevinLoesch1!

**Pull request merged**
